GuiltMotivator/app/src/main/java/nielsen/guiltmotivator/DictionaryOpenHelper.java

```diff
@@ -69,6 +74,14 @@ public void onDowngrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     newTask.setChecked(Boolean.parseBoolean(cursor.getString(cursor.getColumnIndex(DictionaryOpenContract.FeedEntry.COLUMN_NAME_ISCHECKED))));
                     newTask.setId(cursor.getLong(cursor.getColumnIndex(DictionaryOpenContract.FeedEntry._ID)));
 
+                    String dueDateString = cursor.getString(cursor.getColumnIndex(DictionaryOpenContract.FeedEntry.COLUMN_NAME_DUEDATE));
+                    SimpleDateFormat sdf = new SimpleDateFormat("EEE MMM d HH:mm:ss z yyyy");
+                    Date dueDateDate = sdf.parse(dueDateString);
```

Comment:
that variable name though... maybe Calendar should be dueDateCalendar and Date should be dueDate?